nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/ApplyTransformMarkLogic.java

```diff
@@ -92,10 +89,11 @@ public void init(ProcessorInitializationContext context) {
      *
      * @param context
      * @param session
+     * @param incomingAttributes
      * @return
      */
     @Override
-    protected QueryBatchListener buildQueryBatchListener(final ProcessContext context, final ProcessSession session) {
+    protected QueryBatchListener buildQueryBatchListener(final ProcessContext context, final ProcessSession session, Map<String, String> incomingAttributes) {
```

Comment:
There's a separate ticket to "fix" this processor, just had to modify the overridden method for now.